backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/renderInvoiceAddress.feature

```diff
@@ -0,0 +1,248 @@
+@from:cucumber
+@topic:invoiceAddress
+Feature: Render invoice address
+  Especially tracing the "BPartnerName" part of the address
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And metasfresh has date and time 2022-02-25T13:30:13+01:00[Europe/Berlin]
+    And enable sys config 'SKIP_WP_PROCESSOR_FOR_AUTOMATION'
+
+    And metasfresh contains M_Products:
```

Comment:
we also need to cover the case that the customer's location is not in "our" (ad_org's country)